.github/config.yml

```diff
@@ -15,9 +15,13 @@ newPRWelcomeComment: >
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-  Thanks for your contribution to the Layer5 community! :tada:
+  Thanks for your contribution to the Meshery! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/Layer5-celebration.png)
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery-traefik-mesh/master/.github/assets/images/welcome/meshery-welcome-logo-light-text-side.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery-traefik-mesh/master/.github/assets/images/welcome/meshery-welcome-logo-dark-text-side.svg">
```

Comment:
@Amukayoung, wrong name of files in the path

Reply:
Also, update the images with the one I provided

Reply:
Oh sorry, let me handle it thanks

Reply:
I have just updated the logos with a small one of 300 x 51 px and equally corrected their names in the path. Please in case of any further changes, you can let me know. Thanks @thisiskaransgit

Reply:
@Amukayoung, there's still too much extra space identifiable in the image, doesn't look like you have updated them, check again.

Reply:
@thisiskaransgit Thanks for the feedback. Please kindly check the two sets of logos shared because it looks like they are all the same. Actually, I have downloaded and updated the path with the second set of logos (The one you shared here in this conversation)